server/games/game.py

```diff
@@ -80,7 +80,8 @@ class ValidityState(IntEnum):
     FFA_NOT_RANKED = 14
     UNEVEN_TEAMS_NOT_RANKED = 15
     UNKNOWN_RESULT = 16
-
+    MULTI_TEAM = 17
```

Comment:
The database entries for these should be kept in sync

Reply:
you mean I must make a PR to the faf-db aswell ?

Reply:
Yes, as discussed in private.